docker-compose.yml

```diff
@@ -126,6 +126,7 @@ services:
       MERLIN_WORKER_DB_SERVER: postgres
       MERLIN_WORKER_DB_USER: "${AERIE_USERNAME}"
       MERLIN_WORKER_LOCAL_STORE: /usr/src/app/merlin_file_store
+      SIMULATION_EXTENT_MINIMUM_POLL_PERIOD_MILLIS: 2000
```

Comment:
Maybe it's because the word `EXTENT` has now lost all meaning, but `SIMULATION_PROGRESS_REPORTING_FREQUENCY_MILLIS` better explains what the variable is doing from a user-perspective

Reply:
The word progress would work well here :+1: I can make the change.

Frequency and period are not synonyms, however - one is measured in Hertz, and the other in seconds. I think period is a more natural metric to specify here